examples/sample-app/cleanup.sh

```diff
@@ -1,10 +1,15 @@
 #!/bin/sh
 
 echo "Killing openshift all-in-one server ..."
-pkill -x openshift
+sudo pkill -x openshift
+
+echo "Stopping all k8s docker containers on host ..."
+sudo docker ps --format='{{.Names}}' | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r sudo docker stop
```

Comment:
If you start OpenShift with `sudo` but are a member of the `docker` group you do not need `sudo` to stop the containers. 

>  I think that makes more sense than assuming users are members of the docker group.

This is like one of the most basic prerequisites to installing Docker on the system, but if you think that it's usually ignored, then sure, let's use `sudo`.